TABTestKit/Classes/Elements/Attributes.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import CoreGraphics
```

Comment:
Import CoreGraphics because `CGVector` is not available in Foundation

Reply:
Why is this needed?

Reply:
Line 40 of this file

```swift
public typealias NormalizedCoordinate = CGVector
```